### PR TITLE
Query quests & users; basic dialogs for users, quests, feedback

### PR DESCRIPTION
**Pull request by smartin015**
https://github.com/ExpeditionRPG/expedition-admin/issues/1

### CODE REVIEW

File: app/components/Dialogs.tsx [hunk 1]
```diff
@@ -13,26 +12,118 @@ import SelectField from 'material-ui/SelectField'
...
+        title="User"
+        contentClassName="dialog"
+        open={Boolean(this.props.open)}
+        onRequestClose={() => this.props.onRequestClose()}
+        actions={[<FlatButton onTouchTap={() => this.props.onRequestClose()}>Cancel</FlatButton>]}
+      >
+        <p>Name: {this.props.user.name}</p>
+        <p>Email: {this.props.user.email}</p>
+        <p>Loot points: {this.props.user.loot_points}</p>
+        <p>Last login: {this.props.user.last_login.toISOString()}</p>
```
Comment (toddmedema) [inline app/components/Dialogs.tsx]: Might as well also expose login count and created, as they're additional indicators of user activity

File: app/components/Dialogs.tsx [hunk 2]
```diff
@@ -13,26 +12,118 @@ import SelectField from 'material-ui/SelectField'
...
+      <Dialog
+        title="User"
+        contentClassName="dialog"
+        open={Boolean(this.props.open)}
+        onRequestClose={() => this.props.onRequestClose()}
+        actions={[<FlatButton onTouchTap={() => this.props.onRequestClose()}>Cancel</FlatButton>]}
+      >
+        <p>Name: {this.props.user.name}</p>
+        <p>Email: {this.props.user.email}</p>
+        <p>Loot points: {this.props.user.loot_points}</p>
```
Comment (toddmedema) [inline app/components/Dialogs.tsx]: Not sure what you had in mind for editing loot points, but I could either see an edit button in the table cell itself, or in this dialog

File: app/components/Dialogs.tsx (hunk 1 above)
Comment (smartin015) [reply app/components/Dialogs.tsx]: Those aren't exposed in the UserEntry in the api... I added it to the issue.

File: app/components/Dialogs.tsx (hunk 2 above)
Comment (smartin015) [reply app/components/Dialogs.tsx]: Yeah, might just do both. I think the dialog view edit will come first; this is just a framework, so we can add form fields to it in a subsequent PR.

**Pull request merged**